autobot/telegram/parser.py

```diff
@@ -428,7 +429,7 @@ def _parse_inlinequeryresults (self,key:str,val:dict) -> InlineQueryResults:
                         imc = self._parse_inputmessagecontent(k,v)
                         setattr(inline_obj,k,imc)
                     case "reply_markup":
-                        rep = self._parse_inlin
+                        rep = self._parse_inlinekeyboardbuttons(k,v)
```

Comment:
line 432, this is supposed to be `_parse_inlinekeyboardbutton` without an "s" at the end